File: src/main/java/com/company/commitet_jm/service/ones/OneCService.java
package com.company.commitet_jm.service.ones;

import java.io.File;

public interface OneCService {
    void uploadExtFiles(File inputFile, String outDir, String pathInstall, String version);
    void unpackExtFiles(File inputFile, String outDir);
    String pathPlatform(String basePath, String version);
}

File: bin/main/com/company/commitet_jm/service/ones/OneCService.kt
package com.company.commitet_jm.service.ones

import java.io.File

interface OneCService {
    fun uploadExtFiles(inputFile: File, outDir: String, pathInstall: String, version: String)
    fun unpackExtFiles(inputFile: File, outDir: String, pathInstall: String, version: String)
    fun pathPlatform(basePath: String?, version: String?): String
}